x-pack/legacy/plugins/ml/public/application/settings/calendars/edit/calendar_form/calendar_form.js

```diff
@@ -128,7 +133,28 @@ export const CalendarForm = ({
           </EuiFormRow>
         </Fragment>
       )}
-      {isEdit && <EditHeader calendarId={calendarId} description={description} />}
+
+      <EuiFormRow
```

Comment:
Is this label needed at all, as it is just repeating the label of the `EuiSwitch` itself?

![image](https://user-images.githubusercontent.com/7405507/75172083-99dc1d00-5724-11ea-9f55-2a0ef8c4ab5c.png)

Reply:
Fixed in a7802a4837115854613b866b6e7a4b6fcb37e596